services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -108,5 +108,28 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
         projectService.getUserProjectById(projectId, userId).failed.map { _ shouldBe ProjectNotFoundException() }
       }
     }
+
+    "getUserProjectByName" should {
+      val project1 = TestProjectUtils.getDummyProject()
+      val project2 = TestProjectUtils.getDummyProject(name = project1.name)
+
+      "return project with corresponding user id" taggedAs Service in {
+        when(projectRepository.getProjectsByName(project2.name)).thenReturn(Future.successful(Seq(project1, project2)))
+        projectService.getUserProjectByName(project2.name, project2.ownerId).map { _ shouldBe project2 }
```

Comment:
Why do we get only one project, if we recive sequence of two projects ?

Reply:
I think we should get only one project at Services level whereas at level of Repository we get sequence of projects.